test/url-to-options.js

```diff
@@ -0,0 +1,56 @@
+import url from 'url';
+import test from 'ava';
+import urlToOptions from '../source/url-to-options';
+
+const githubURL = 'https://alex:pass@github.com:443/';
```

Comment:
`const githubURL = 'https://user:password@github.com:443/';`

Reply:
Sorry for letting my ego creep into the code 🙈 .